src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/my profile/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] It looks like you have only one test, and I think this is the default project test when you create the project. Please, add tests for your project, this is an important part of the project. Try to follow the test guidelines for your projects before this and do it as completely as possible. 🐱